### DISCUSSION TITLE
Tutorial for dummies please?

### DISCUSSION THREAD

**Discussion by juanpeich** [giejay/Immich-Android-TV#46] [Q&A]
Could you give a tutorial for dummies like me?

I have never handled code and I would like to install it on my FireStick 4k Max.

I have followed the first step, but I don't know if it should be done via the command line within the raspberry where I have Immich installed, and I also don't know which option I should give within Google Firebase.

Thank you very much in advance and sorry for the inconvenience

**Comment by giejay** [comment]
You can simply sideload it on your FireStick. Download it from the releases tab and then use Send Files to TV to send the APK.

**Reply by giejay** [reply]
https://github.com/giejay/Immich-Android-TV/releases

**Reply by juanpeich** [reply]
Thank You so much for your quickly answer

**Reply by devtechk** [reply]
> You can simply sideload it on your FireStick. Download it from the releases tab and then use Send Files to TV to send the APK.

The apk is using firebase and  Which account it use? Thank you!

**Comment by devtechk** [comment]
Hello same here, I'm notable to set firebase as wirtten in the readme file. I think it miss the android app creation from firebase. Can you do a step by step guide or more detailed one? Thank you!

**Reply by juanpeich** [reply]
I have already received a reply to download the APK directly and install it. I have it working perfectly now.

https://github.com/giejay/Immich-Android-TV/releases
